copulas/multivariate/gaussian.py

```diff
@@ -93,15 +93,12 @@ def func(*args):
 
     def sample(self, num_rows=1):
         res = {}
-
-        # clean up means
-        clean_mean = np.nan_to_num(self.means)
+        means = [0] * len(self.means)
```

Comment:
This could be replaced with `np.zeros(len(self.means)))`, that it's more clear than multiplying a list.